### PR TITLE
A handfull of CAP stuff

### PR DESCRIPTION
**Pull request by not-the-fiend**
Adds Bulbfrog, Ballboa, Semmush, Mega Hohohoming, Mega Faptime, Mega Jerkle, Mega Dowster and Mega Reptrill
Adds Spooky Aura
Adds Livewire, Dragon Burst and Rock Clock
Nerfs Mega Krokizon

### CODE REVIEW

**Comment by not-the-fiend** [comment]
ALSO ADDS PSYCHO FISTS TO HUSBIN'S LEARNSET WHICH I FORGOT TO DO WHEN I FINISHED CODING IT IN

File: data/items.ts [hunk 1]
```diff
@@ -7835,6 +7835,81 @@ export const Items: { [itemid: string]: ItemData } = {
...
+		num: -1,
+		gen: 8,
+		isNonstandard: "Future",
+	},
+	reptrillite: {
+		availability: {clover: 1},
+		name: "Reptrillite",
+		spritenum: 577,
+		megaStone: "Reptrill-Mega",
+		megaEvolves: "Reptrille",
```
Comment (MrSableye) [inline data/items.ts:7903]: nice spelling

**Comment by MrSableye** [comment]
Can you add format data for `data/formats.ts` and `data/mods/clover/formats.ts`?
As well as enabling the new items in `data/mods/clover/items.ts` and enabling the new moves in `data/mods/clover/moves.ts`?

**Comment by not-the-fiend** [comment]
ok sure

**Review by MrSableye — APPROVED** [review]

**Pull request merged**